src/views/PancakeSquad/components/Header/index.tsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { Box, Flex, lightColors, Spinner, Text, Timeline } from '@pancakeswap/uikit'
+import { useTranslation } from 'contexts/Localization'
+import { useGetCakeBalance } from 'hooks/useTokenBalance'
+import useTheme from 'hooks/useTheme'
+import { StyledWaveContainer } from 'views/PancakeSquad/styles'
+import { formatBigNumber } from 'utils/formatBalance'
+import { UserStatusEnum } from 'views/PancakeSquad/types'
+import ConnectWalletButton from 'components/ConnectWalletButton'
+import HeaderBottomWave from '../../assets/HeaderBottomWave'
+import nftSaleConfigBuilder from '../../config'
+import CtaButtons from './CtaButtons'
+import MintText from './MintText'
+import PreEventText from './PreEventText'
+import SaleProgress from './SaleProgress'
+import {
+  StyledSquadEventBorder,
+  StyledSquadEventContainer,
+  StyledSquadHeaderContainer,
+  StyledSquadTitle,
+} from './styles'
+import { PancakeSquadHeaderType } from './types'
+
+const DEFAULT_CAKE_COST = 5
+const DEFAULT_MAX_TICKETS = 10
+
+const PancakeSquadHeader: React.FC<PancakeSquadHeaderType> = ({
+  userInfos,
+  eventInfos,
+  account,
+  userStatus,
+  isLoading,
+}) => {
+  const { t } = useTranslation()
+  const { theme, isDark } = useTheme()
+  const { balance: cakeBalance } = useGetCakeBalance()
+  const hasData = !!eventInfos && !!userInfos
+  const displayEventBlock = !!eventInfos || isLoading
+  const {
+    ticketsOfUser,
+    numberTicketsUsedForGen0,
+    numberTicketsOfUser,
+    numberTicketsForGen0,
+    canClaimForGen0,
+    numberTokensOfUser,
+  } = userInfos || {}
+
+  const {
+    maxPerAddress,
+    maxPerTransaction,
+    maxSupply,
+    pricePerTicket,
+    saleStatus,
+    totalTicketsDistributed,
+    totalSupplyMinted,
+    startTimestamp,
+  } = eventInfos || {}
+
+  return (
+    <StyledSquadHeaderContainer
+      pt={['16px', null, null, '40px']}
+      px={['16px', null, null, '80px']}
+      flexDirection="column"
+      alignItems="center"
+    >
+      <Flex width="100%">
+        <Link to="/nfts">
+          <Text color="primary" bold>{`< ${t('NFT Market')}`}</Text>
+        </Link>
+      </Flex>
+      <StyledSquadTitle my="32px" color={lightColors.invertedContrast} bold textAlign="center">
+        {t('Pancake Squad')}
+      </StyledSquadTitle>
+      <Text color={lightColors.warning} textAlign="center" bold>
+        {t('Mint Cost: To Be Announced', {
+          minCost: pricePerTicket ? formatBigNumber(pricePerTicket, 0) : DEFAULT_CAKE_COST,
+        })}
+        <br />
```

Comment:
Usually in other components we don't use `br` and just wrap text into multiple `<Text>` components. Sorry for nitpicks 😅

Reply:
Can do, no problem.
Using `<br />` in place of a react component might be better for performance in this case though.